ecosystem/sep-0057.md

```diff
@@ -0,0 +1,1356 @@
+## Preamble
+
+```
+SEP: 0057
+Title: T-REX (Token for Regulated EXchanges)
+Author: OpenZeppelin, Boyan Barakov <@brozorec>, Özgün Özerk <@ozgunozerk>, Dennis O'Connell <@droconnel22>
+Status: Draft
+Created: 2025-11-26
+Updated: 2025-11-26
+Version: 0.1.0
+Discussion: https://github.com/orgs/stellar/discussions/1814
+```
+
+## Summary
+
+This proposal defines a standard contract interface for T-REX (Token for
+Regulated EXchanges) - colloquially known as RWA token - on Stellar. T-REX
+tokens represent tokenized real-world assets such as securities, bonds, real
+estate, or other regulated financial instruments that require compliance with
+regulatory frameworks. T-REX tokens are **permissioned tokens**, ensuring that
+only eligible investors can hold and transfer these tokens.
```

Comment:
ensuring secure and compliant transactions for all parties involved in the token transfer.